<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogSettings</class>
 <widget class="QWidget" name="LogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>466</height>
   </rect>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>340</width>
     <height>106</height>
    </rect>
   </property>
   <property name="title">
    <string>Export data during logging</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkWriteDuringLog">
      <property name="text">
       <string>Save data to temporary file while logging</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <item>
       <widget class="QLineEdit" name="lineWriteDuringLogFile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonWriteDuringLogFile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>121</y>
     <width>523</width>
     <height>138</height>
    </rect>
   </property>
   <property name="title">
    <string>Time data</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Poll time (ms)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinPollTime">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Time reference</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioRelative">
        <property name="text">
         <string>Relative (use start of log as time reference)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioAbsolute">
        <property name="text">
         <string>Absolute (use current time/date as time reference)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>checkWriteDuringLog</tabstop>
  <tabstop>lineWriteDuringLogFile</tabstop>
  <tabstop>buttonWriteDuringLogFile</tabstop>
  <tabstop>spinPollTime</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
